### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by dg0yt**

### CODE REVIEW

File: .github/workflows/ci_ubuntu.yml [hunk 1]
```diff
@@ -192,14 +191,13 @@ jobs:
...
           export TMP=${{runner.temp}}
-          echo "TMP=${{github.temp}}" >> $GITHUB_ENV
           CTEST_OUTPUT_ON_FAILURE=1 ctest -C ${{env.BUILD_TYPE}}
 
       - name: Configure and Make
         working-directory: ${{github.workspace}}/
         run: |
           ./bootstrap.sh
-          ./configure --with-png --with-jpeg --with-ramq --with-heif --with-xpm --with-tiff --with-webp --with-liq --enable-gd --enable-gd-formats --with-zlib
+          ./configure --disable-werror --with-png --with-jpeg --with-ramq --with-heif --with-xpm --with-tiff --with-webp --with-liq --enable-gd --enable-gd-formats --with-zlib
```
Comment (vapier) [inline .github/workflows/ci_ubuntu.yml:200]: we def do not want to disable -Werror in the CI

**Review by vapier — CHANGES_REQUESTED** [review]
please describe in detail what/why you're changing things in the respective commit messages.  just writing "update" doesn't help anyone review or understand what you're trying to do.

**Comment by dg0yt** [comment]
> please describe in detail what/why you're changing things in the respective commit messages. just writing "update" doesn't help anyone review or understand what you're trying to do.

Feel free to cherry-pick and to amend. I re-synced #851, and this is all I want to offer at the moment.

**Comment by vapier** [comment]
considering CI is passing without your changes, and there's no explanation as to what you're trying to do, the default inclination would be to not merge anything.  if you're not interested in addressing feedback, then we can close the PR.

**Comment by dg0yt** [comment]
> CI is passing without your changes

Is it?

https://github.com/libgd/libgd/pull/851/checks?check_run_id=19030643780

Windows Mingw CI is broken
macOS CI is broken
Ubuntu CI is broken

**Comment by vapier** [comment]
the Ubuntu CI static warnings are fixed.

macOS CI shows green at the top-level.  if it isn't actually green, that probably should be fixed.  maybe you're fixing that here ... i have no idea because you haven't explained anything.

if your PR is addressing something on the latest master branch, then it should be easy to describe in your commit messages what you're fixing.  i don't understand why this is an issue for you ... this is completely standard git flow.  if you're new to git, please check out this page:
https://cbea.ms/git-commit/

**Comment by dg0yt** [comment]
Sure, macOS is green at the "top-level". 
https://github.com/libgd/libgd/actions/workflows/ci_macos.yml

And it is dark red on the job level, e.g. 
https://github.com/libgd/libgd/actions/runs/7015560144/job/19085086520

I may be able to edit the commit messages when I eventually do a git checkout. But you may not want to wait for it.

**Pull request closed without merging**